cllc-public-app/ClientApp/src/app/file-uploader/file-uploader.component.ts

```diff
@@ -77,6 +77,12 @@ export class FileUploaderComponent implements OnInit {
       alert('File type not supported.');
       return;
     }
+
+    if(file && file.name && file.name.length > 60){
+      alert('File name must be 50 characters or less.');
```

Comment:
1. 60 is not 50.
2. Constraints like this should be stored in a constant.  In this case it should go in the SharePoint File Manager class.
3. The actual limit for SharePoint is 128 characters for a file and 260 for a URL path (in on prem SharePoint)

https://support.office.com/en-us/article/invalid-file-names-and-file-types-in-onedrive-onedrive-for-business-and-sharepoint-64883a5d-228e-48f5-b3d2-eb39e07630fa